platform/core/src/classes/index.js

```diff
@@ -18,16 +18,14 @@ import { StudyMetadataSource } from './StudyMetadataSource';
 import { StudyPrefetcher } from './StudyPrefetcher';
 import { TypeSafeCollection } from './TypeSafeCollection';
 
-//import { StudySummary } from './metadata/StudySummary';
-
 export {
   OHIFStudyMetadataSource,
   MetadataProvider,
   CommandsManager,
   HotkeysManager,
   ImageSet,
   StudyPrefetcher,
-  //StudyLoadingListener,
```

Comment:
I'm not familiar with what this was for or did/does

Reply:
Monitoring the progress of image loading across the study. We have to figure out how this works with the change to an extension-based architecture.

Reply:
I agree that it would be good to have it back.